frontend/catalyst/passes/builtin_passes.py

```diff
@@ -669,3 +669,114 @@ def circuit():
         )
 
     return PassPipelineWrapper(qnode, passes)
+
+
+def to_ppm(
+    qnode=None, *, decompose_method="auto-corrected", avoid_y_measure=False, max_pauli_size=0
+):
+    R"""
+    Specify that the MLIR compiler pass for transforming
+    clifford+T gates into Pauli Product Measurements (PPM) will be applied.
+
+    This pass combines multiple sub-passes:
+
+    - :func:`~.passes.to_ppr` : Converts gates into Pauli Product Rotations (PPRs)
+    - :func:`~.passes.commute_ppr` : Commutes PPRs past non-Clifford PPRs
+    - :func:`~.passes.merge_ppr_ppm` : Merges PPRs into Pauli Product Measurements (PPMs)
+    - :func:`~.passes.ppr_to_ppm` : Decomposes PPRs into PPMs
+
+    The ``avoid_y_measure`` and ``decompose_method`` arguments are passed
+    to the :func:`~.passes.ppr_to_ppm` pass.
+    The ``max_pauli_size`` argument is passed to the :func:`~.passes.commute_ppr`
+    and :func:`~.passes.merge_ppr_ppm` passes.
+
+    Args:
+        qnode (QNode, optional): QNode to apply the pass to. If None, returns a decorator.
+        decompose_method (str, optional): The method to use for decomposing non-Clifford PPRs.
+            Options are ``"auto-corrected"`` and ``"clifford-corrected"``. Defaults to
+            Defaults to ``"auto-corrected"``.
+            ``"auto-corrected"`` uses an additional measurement for correction.
+            ``"clifford-corrected"`` uses a Clifford rotation for correction.
+        avoid_y_measure (bool): Rather than performing a Pauli-Y measurement for Clifford rotations
+            (sometimes more costly), a :math:`Y` state (:math:`Y\vert 0 \rangle`) is used instead
+            (requires :math:`Y` state preparation). Defaults to ``False``.
+        max_pauli_size (int): The maximum size of the Pauli strings after commuting or merging.
+            Defaults to 0 (no limit).
+
+    Returns:
+        ~.QNode or callable: Returns decorated QNode if qnode is provided,
+            otherwise returns a decorator.
```

Comment:
The extra indentation here puts the first and second lines on separate paragraphs.

```suggestion
        ~.QNode or callable: Returns decorated QNode if qnode is provided,
        otherwise returns a decorator.
```